modules/wkt/src/lib/parse-wkb.ts

```diff
@@ -23,19 +31,13 @@ export default function parseWKB(buffer) {
   switch (geometryType) {
     case 1:
       const point = parsePoint(view, offset, dimension, littleEndian);
-      delete point.offset;
```

Comment:
I don't remember why I thought I needed to delete the `offset` key from the geometry.